src/compiler/types.ts

```diff
@@ -703,6 +704,7 @@ module ts {
 
     export interface Statement extends Node, ModuleElement {
         _statementBrand: any;
+        downlevelTaggedTemplates?: TaggedTemplateExpression[]; // Initialized in binder.ts

```

Comment:
This really does't seem appropriate; I believe we manage to get emit for destructuring without tacking a property on any other node.